_common

```diff
@@ -171,7 +171,6 @@ handle_unreviewed() {
     local testurl=$1 out=$2 reason=$3 group_id=$4 email_unreviewed=$5 from_email=$6 notification_address=${7:-} job_data=${8:-} dry_run=${9:-}
     local group_data group_name group_description group_mailto header email excerpt=$snip_start
     local job_name job_result info
-    to_review+=("$testurl ${reason:0:50}")
```

Comment:
https://github.com/os-autoinst/scripts/blob/37be6df7f557ec0f5b43e45f77665cb30e792ff6/openqa-label-known-issues-multi#L16

Reply:
How about ensuring this is covered in a test before making this change?

Reply:
To me it looks like we don't want this change at all because the variable is in fact used.

Reply:
Nice catch @baierjan 
I thought we were using a bash option to prevent undeclared variables but I guess it doesn't work that way here...

Reply:
If I am not mistaken https://github.com/os-autoinst/scripts/blob/37be6df7f557ec0f5b43e45f77665cb30e792ff6/openqa-label-known-issues-multi#L10 counts as a declaration 😄 

The catch is, that the variable is filled as a kind of "side-effect" of `label_issue` call. Fortunately, it is still kind of clear once you try `git grep` for that variable.

Reply:
But when this is called by `openqa-label-known-issues` and not by `scripts/openqa-label-known-issues-multi` there is no initial declaration.

But now I know - in `openqa-investigate ` for example we are using `-u` (`set -euo pipefail`)- but not in `openqa-label-known-issues`.
Adding `-u` there should make it fail.

Reply:
Right, other scripts sourcing `_common` are missing the declaration.